admin/class-spid-admin.php

```diff
@@ -1,249 +1,257 @@
-        {
-            add_action('admin_menu', array($this, 'spid_settings_menu'));
-            add_action('admin_init', array($this, 'spid_options_init'));
-        }
-
-        public function spid_settings_menu()
-        {
-            add_menu_page('SPID Opzioni', 'SPID Opzioni', 'manage_options', 'spid_opzioni', array($this, 'spid_admin_options'));
-        }
-
-        function spid_admin_options()
-        {
-            $this->spid_options_general = get_option('spid_general');
-            $this->spid_options_button = get_option('spid_button');
-            $this->spid_options_help = get_option('spid_help');
-
-            $button_Screen = (isset($_GET['action']) && 'button' == $_GET['action']) ? true : false;
-            $help_Screen = (isset($_GET['action']) && 'help' == $_GET['action']) ? true : false;
-
-            ?>
-            <div class="wrap">
-            <h1>SPID Options</h1>
-            <h2 class="nav-tab-wrapper">
-                <a 
-                    href="<?php echo admin_url('admin.php?page=spid_opzioni'); ?>"
-                    class="nav-tab<?php if (!isset($_GET['action']) || isset($_GET['action']) && 'button' != $_GET['action'] && 'help' != $_GET['action']) echo ' nav-tab-active'; ?>">
-                    <?php esc_html_e('General'); ?>
-                </a>
-                <a 
-                    href="<?php echo esc_url(add_query_arg(array('action' => 'button'), admin_url('admin.php?page=spid_opzioni'))); ?>" 
-                    class="nav-tab<?php if ($button_Screen) echo ' nav-tab-active'; ?>">
-                    <?php esc_html_e('SPID Button'); ?></a> 
-                <a 
-                    href="<?php echo esc_url(add_query_arg(array('action' => 'help'), admin_url('admin.php?page=spid_opzioni'))); ?>" 
-                    class="nav-tab<?php if ($help_Screen) echo ' nav-tab-active'; ?>">
-                    <?php esc_html_e('Help'); ?>
-                </a>        
+if ( ! class_exists( 'SPID_Admin' ) ) {
+	class SPID_Admin {
+
+		private $spid_options_general;
+		private $spid_options_button;
+		private $spid_options_help;
+
+		public function __construct() {
+			add_action( 'admin_menu', array( $this, 'spid_settings_menu' ) );
+			add_action( 'admin_init', array( $this, 'spid_options_init' ) );
+		}
+
+		public function spid_settings_menu() {
+			add_menu_page( 'SPID Opzioni', 'SPID Opzioni', 'manage_options', 'spid_opzioni', array( $this, 'spid_admin_options' ) );
+		}
+
+		function spid_admin_options() {
+			$this->spid_options_general = get_option( 'spid_general' );
+			$this->spid_options_button  = get_option( 'spid_button' );
+			$this->spid_options_help    = get_option( 'spid_help' );
+
+			$button_Screen = ( isset( $_GET['action'] ) && 'button' == $_GET['action'] ) ? true : false;
+			$help_Screen   = ( isset( $_GET['action'] ) && 'help' == $_GET['action'] ) ? true : false;
+
+			?>
+			<div class="wrap">
+			<h1>SPID Options</h1>
+			<h2 class="nav-tab-wrapper">
+				<a 
```

Comment:
the formatting applied to the HTML code here seems redundant

Reply:
If that part was made by a human, I accept it. I assumed it was the autoformatter !